### ISSUE TITLE
cannot resolve method.

### ISSUE THREAD

**Issue by phoenixstudio** [dexafree/MaterialList#50]
Hello, thanks for this great library.
 
I would like to request some help. I've followed the steps, but I still can't add the card to the Material List.
I've added the "compile 'com.github.dexafree:materiallist:2.3.0'" to dependencies.
![screen shot 2015-03-02 at 2 06 45 am](https://cloud.githubusercontent.com/assets/1477874/6432163/d65761e6-c083-11e4-935b-580264e62297.png)
and also this error while in the preview page.
![screen shot 2015-03-02 at 2 06 31 am](https://cloud.githubusercontent.com/assets/1477874/6432175/00115e24-c084-11e4-9ca1-f93a07101199.png)
how to solve?
thanks.

**Comment by dexafree** [comment]
Hi!

Instead of

```java
Card card = new BasigImageButtonsCard(this);
```

Try by

```
BasicImageButtonsCard card = new BasicImageButtonsCard(this);
```

`Card` is the most abstract type, made to represent any `Card`, so there might exist a Card which don't have a title.
The first card that has the title attribute is `SimpleCard` (https://github.com/dexafree/MaterialList/blob/master/materialList/src/main/java/com/dexafree/materialList/cards/SimpleCard.java), so if you want to create Cards that have a title on it, the minimum inheritance level you should use is `SimpleCard`

Hope I explained well!

**Comment by phoenixstudio** [comment]
Great.
My memory about inheritance comes back!
Thanks and sorry for troubling you. 
The documentation might a bit different due to code changes?
I hope I can contribute.

**Comment by dexafree** [comment]
Yes, I will have to take a look to the docs, as the API has experienced some changes since it was done
